### ISSUE TITLE
`Style/SingleArgumentDig`: false positive for arguments forwarding

### ISSUE THREAD

**Issue by yujideveloper** [rubocop/rubocop#10574]
## Expected behavior

It does not detect offense of `Style/SingleArgumentDig`.

## Actual behavior

```
$ rubocop --debug --cache false --only Style/SingleArgumentDig foo.rb
For /Users/yuji_developer/foo: configuration from /Users/yuji_developer/foo/.rubocop.yml
Default configuration from /Users/yuji_developer/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/rubocop-1.28.1/config/default.yml
Inspecting 1 file
Scanning /Users/yuji_developer/foo/foo.rb
C

Offenses:

foo.rb:7:5: C: [Correctable] Style/SingleArgumentDig: Use @hash[...] instead of @hash.dig(...).
    @hash.dig(...)
    ^^^^^^^^^^^^^^

1 file inspected, 1 offense detected, 1 offense auto-correctable
Finished in 0.05598199996165931 seconds

```

## Steps to reproduce the problem

Run `rubocop --only Style/SingleArgumentDig foo.rb` on the following config and code.

```yaml
# .rubocop.yml
AllCops:
  TargetRubyVersion: 2.7
  NewCops: disable
```
```ruby
# foo.rb
class Foo
  def initialize
    @hash = {}
  end

  def dig(...)
    @hash.dig(...)
  end
end

```

## RuboCop version

```
$ rubocop -V
1.28.1 (using Parser 3.1.2.0, rubocop-ast 1.17.0, running on ruby 2.7.6 arm64-darwin21)
```
